upp/stages/resampling.py

```diff
@@ -141,9 +141,22 @@ def run_on_region(self, components, region):
 
         # groupby samples
         for sample, cs in components.groupby_sample():
+            # make sure all tags equal_jets_from_samples are the same
+            equal_jets_flags = [c.equal_jets_from_samples for c in cs]
+            if not all(flag == equal_jets_flags[0] for flag in equal_jets_flags):
+                raise ValueError(
+                    "equal_jets_from_samples must be the same for all components in a sample"
+                )
+            equal_jets_flag = equal_jets_flags[0]
```

Comment:
Is this check required? The `equal_jets_from_samples` is added from the sample in the config right? In any case I guess it doesn't hurt to have it

Reply:
I've added the flag to the `sample` part of the config, e.g.
```
hbb: &hbb
  name: hbb
  pattern: path/to/sample/*.h5
  equal_jets_from_samples: False
```
I've added this check just in case, as `groupby_sample()` should return a list of components with the same sample. This might be irrelevant here, but if we have a list of `equal_jets` flags it can't run unless all the flags are the same, might be worth checking.